src/core/orderbook.js

```diff
@@ -258,6 +258,8 @@ OrderBook.prototype.unsubscribe = function() {
 OrderBook.prototype.requestOffers = function(callback) {
   const self = this;
 
+  var callback = typeof callback === 'function' ? callback : function(){};
```

Comment:
This redefines a symbol, which is forbidden by our lint rules. You can use es6 default parameters instead: `function(callback=function() {})`